app/controllers/federations_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+##
+# Provides Federated content via JSON, to be displayed within a content Collection
+class FederationsController < ApplicationController
+  # The federated action which is used to retrieve federated results via the foederati gem.
+  def show
+    @collection = find_collection
+    provider = params[:id]
+    federation_config = @collection.federation_configs.where(provider: provider).first
+    @query = "#{params[:query]} #{federation_config.context_query}"
```

Comment:
empty searches are a bit problematic (specifically for trove/dpla) when combined with a "context" I'm trying to figure out a better solution for this.